docs/addons/amp-addon.md

```diff
@@ -88,30 +94,35 @@ import * as cdk from 'aws-cdk-lib';
 import * as blueprints from '@aws-quickstart/eks-blueprints';
 
 const app = new cdk.App();
+const ampWorkspaceName = "blueprints-amp-workspace";
+const ampPrometheusEndpoint = (blueprints.getNamedResource(ampWorkspaceName) as unknown as amp.CfnWorkspace).attrPrometheusEndpoint;
```

Comment:
This should be changed at least in the docs. 

I suggest this syntax:

```typescript
const ampWorkspace: CfnWorkspace = blueprints.getNamedResource(ampWorkspaceName);

const addOn = new blueprints.addons.AmpAddOn({
    ampPrometheusEndpoint: ampWorkspace.attrPrometheusEndpoint,
 ..
```

I saw that you struggled with the data types. Please pull from my feature/cfn-resource-providers branch first, it has a change that will make it happen.

Reply:
Fixed. great suggestion and also on the cfn PR.